sql/types/json_encode.go

```diff
@@ -8,6 +8,27 @@ import (
 	"strconv"
 )
 
+var isEscaped = [256]bool{}
```

Comment:
Could combine these two and use `esc, ok := escapeSeq[r]` checks, any reason not to?

Reply:
I thought it would be slower.  I could be wrong.

Reply:
That requires a map instead of an array.

Reply:
That requires a map instead of an array.